pyproject.toml

```diff
@@ -141,6 +141,10 @@ line-ending = "auto"
 [tool.pyright]
 exclude = ["mirascope/beta"]
 
+[tool.coverage.report]
+exclude_also = [
+    "@overload"
+]
```

Comment:
we can ignore the coverage in `@overload`

Reply:
amazing